cli/kubectl-hyperkube

```diff
@@ -288,5 +299,19 @@ def _load_pem(pem):
         sys.exit(1)
 
 
+def write_kubeconfig(fetched_kubeconfig, destination_file):
+  temp_kubeconfig_file = "temp-kubeconfig.yaml"
+  temp_file = open(temp_kubeconfig_file, 'w+')
+  yaml.dump(fetched_kubeconfig, temp_file, allow_unicode=True)
+  
+  if os.path.exists(destination_file):
+    source_kubeconfigs = [temp_kubeconfig_file, destination_file]
+  else:
+    source_kubeconfigs = [temp_kubeconfig_file]
+
+  merge(source_kubeconfigs, destination_file)
```

Comment:
Are there any side effects with merging configuration? Can this be an option? The default should probably be overwrite like previous versions of hyperkube with an option to merge.

Reply:
Previous versions of hyperkube never wrote anything to the filesystem. In almost all the cases ,users are redirecting the output to a file. You can still do that same thing by passing `-s` flag and then redirecting the output.

Reply:
Ok cool.

Reply:
See my comment above about backwards compatibility. Is the plan just to break it and make sure users make the appropriate changes to their scripts?

Reply:
addressed.